language/documentation/tutorial/README.md

```diff
@@ -41,9 +41,9 @@ Follow the prompt to install all the necessary dependencies.
 ```bash
 $ . ~/.profile
 ````
-- Install Move CLI by running this command in diem repo:
+- Install Move's CLI's by running this command in diem repo:
```

Comment:
Can't be CLI's because it's not possessive.
1. Plural? `CLIs`
2. Singular "Install Move's CLI"

Reply:
I see, thanks! I'll change this.

```suggestion
- Install Move's CLIs by running this command in diem repo:
```